WeatherForecastApp/Controller/LocationDetailViewController.swift

```diff
@@ -7,13 +7,15 @@
 import Foundation
 import UIKit
 import GooglePlaces
+import CoreLocation
 
 private let dateFormatter: DateFormatter = {
```

Comment:
Global functions are generally a bad idea, OOP principals and clean code guidelines should be used here. Encapusulate this functionality in its own struct or class that deals with date. You could maybe name it DateUtility.swift